api/swagger.yml

```diff
@@ -4993,6 +5002,51 @@ paths:
         default:
           $ref: "#/components/responses/ServerError"
 
+  /repositories/{repository}/branches/{branch}/objects/stat/user_metadata:
+    parameters:
+      - in: path
+        name: repository
+        required: true
+        schema:
+          type: string
+      - in: path
+        name: branch
+        required: true
+        schema:
+          type: string
+        description: branch to update
+      - in: query
+        name: path
+        description: path to object relative to the branch
+        required: true
+        schema:
+          type: string
+    put:
+      tags:
+        - objects
+        - experimental
+      operationId: updateObjectUserMetadata
+      summary: update object metadata
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/UpdateObjectUserMetadata"
+      responses:
+        201:
```

Comment:
Is 201 Created customary for a PUT that updates an existing resource? (not asking that you'd change it, legitimately asking!)

Reply:
[MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT#successfully_creating_a_resource) says we're good:
> If the target resource does have a current representation and that representation is successfully modified with the state in the request, the origin server must send either a [200 OK](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200) or a [204 No Content](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204) to indicate successful completion of the request

Here we're always on the modification flow, you cannot PUT this is the object doesn't exist.